pkg/image/apiserver/registry/imagestream/etcd/image.go

```diff
@@ -231,3 +231,7 @@ func (l *ImageLayers) GetOwnerReferences() []metav1.OwnerReference       { retur
 func (l *ImageLayers) SetOwnerReferences([]metav1.OwnerReference)        {}
 func (l *ImageLayers) GetClusterName() string                            { return "" }
 func (l *ImageLayers) SetClusterName(clusterName string)                 {}
+func (l *ImageLayers) GetManagedFields() []metav1.ManagedFieldsEntry {
+	return []metav1.ManagedFieldsEntry{}
```

Comment:
What do kube resources do here? Is this a TODO for 1.15 or 1.16?

Reply:
This is just a implementation to fulfill interface.